src/agent/at-driver.js

```diff
@@ -13,19 +14,23 @@ import { iterateEmitter } from '../shared/iterate-emitter.js';
 export async function createATDriver({
   url: { hostname = 'localhost', port = 4382 } = {},
   abortSignal,
+  log,
 } = {}) {
   if (!abortSignal) process.exit(1);
-  const socket = new ws(`ws://${hostname}:${port}`, ['v1.aria-at.bocoup.com']);
-  const driver = new ATDriver({ socket });
+  const socket = new ws(`ws://${hostname}:${port}/command`);
```

Comment:
also - I don't know if this `/command` is needed, but I'm realizing that the at-driver url passed in via command line arguments is an http, which doesn't make sense.  Should we change that parameter name/usage to pass the full websocket url instead?

Reply:
[The proposal specifies "/session",](https://w3c.github.io/at-driver/#transport) so this may be a bug in the plugin.

I'm also realizing that the specification doesn't comment on the `Sec-WebSocket-Protocol` header. For the time being, it's fine that the plugin doesn't require it, but I'm going to open an issue against AT Driver to propose formalizing that part.

Reply:
> [...] I'm realizing that the at-driver url passed in via command line arguments is an http, which doesn't make sense.  Should we change that parameter name/usage to pass the full websocket url instead?

(Sorry, I missed this part in my earlier response.)

It makes a little sense :P [The WebSocket protocol starts with a handshake sent via HTTP(S).](https://datatracker.ietf.org/doc/html/rfc6455#section-4.2.1) We may give the WebSocket constructor a URL with a scheme like `ws:` or `wss:`, but [it constructs an HTTP(S) request for the handshake based on that URL](https://github.com/websockets/ws/blob/7460049ff0a61bef8d5eda4b1d5c8170bc7d6b6f/lib/websocket.js#L715).

Accepting a URL seems like an ergonomic way of allowing folks to specify the protocol, host, and port. I'm okay with accepting HTTP(S) protocols in addition to WS(S), given the particulars described above. As it stands, though, the harness appears to be ignoring the specified protocol entirely. I'd like to address [that bug](https://github.com/w3c/aria-at-automation-harness/issues/18) in a follow-up patch.

Reply:
Updated to use the `pathname` passed in from the `--agent-at-driver-url` param - It is currently forcing the `ws` protocol regardless of which protocol you define on the command line - could look into adding "s" support but I think this is probably best handled in that followup